### DISCUSSION TITLE
"Save and add another" for fields, and group addition during new/edit field flow

### DISCUSSION THREAD

**Discussion by mckenzieclark** [craftcms/cms#13865] [Ideas] [answered]
When building out a CMS I tend to find myself adding a bunch of fields in quick succession and wanting to 'save and add another' as can be done with Entries. Does anyone else find it a tad frustrating not being able to do this?

Another bit of unnecessary back n forth I find is with field groups, and having to assign a field to a group after the fact, if the group hasn't already been created.

This mockup should indicate where I'm going with this.

<img width="997" alt="field-features" src="https://github.com/craftcms/cms/assets/25327/483ca4a0-371e-490d-b71c-abc78ce70364">

Couple of other shouts here:
https://github.com/craftcms/cms/discussions/9614
https://github.com/craftcms/cms/discussions/13643

**Comment by brandonkelly** [comment, marked as answer]
Thanks for the suggestion! I’ve added a “Save and add another” action for Craft 4.6 (f11b1ecf63c2411b53fecabe3bd6c3f2b4f54cb2).

Field groups are going away in Craft 5, so adding inline field group creation wouldn’t be worth the effort.

**Reply by brandonkelly** [reply]
Craft 4.6.0 is out with that change 🎉
